x-pack/heartbeat/seccomp_linux.go

```diff
@@ -61,23 +117,16 @@ func init() {
 			"sendto",
 			"set_robust_list",
 			"set_tid_address",
-			"si_code",
 			"sigaltstack",
-			"SIGINT",
-			"SIGURG",
-			"SI_KERNEL",
-			"si_pid",
-			"si_signo",
-			"SI_TKILL",
-			"si_uid",
 			"socket",
 			"umask",
 			"uname",
-			"with",
-			"write"}
+			"write",
+		}
 
 		if err := seccomp.ModifyDefaultPolicy(seccomp.AddSyscall, syscalls...); err != nil {
 			panic(err)
 		}
+		fmt.Printf("Installed seccomp policy")
```

Comment:
Will this show up in the logs or only stdout?

Reply:
Removed this in a new refactor. You can't use logger from init, but it's actually logged elsewhere so is unnecessary.